distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java

```diff
@@ -171,176 +166,151 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void testCswAfterDateQuery()
-            throws XPathException, ParserConfigurationException, SAXException, IOException {
+    public void testCswAfterDateQuery() throws Exception {
```

Comment:
Many test methods must now declare an Exception. The new XSearch helper class does not swallow exceptions because that can make debugging more difficult.